wfe2/wfe_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"golang.org/x/crypto/ocsp"
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/types/known/emptypb"
-	jose "gopkg.in/go-jose/go-jose.v2"
+	"gopkg.in/go-jose/go-jose.v2"
```

Comment:
Looks like a result of fmt on save? This is usually fine, but this file has no other changes.

Reply:
Yeah, I think I touched this file while experimenting, hence the change.  That import alias is redundant so I just committed it anyways.

Reply:
It's widespread so I'll revert that for now and we can do a seperate cleanup PR